packages/stylelint-config/README.md

```diff
@@ -60,11 +60,90 @@ export default {
 ```json
 {
   "stylelint": {
-    "extends": ["@almacareer/stylelint-config", "stylelint-config-prettier"]
+    "extends": ["@almacareer/stylelint-config", "stylelint-prettier/recommended"]
   }
 }
 ```
 
 </details>
 
----
+## Checking Properties Order
+
+To further extend control over coding style of your stylesheets, we are also
+checking for properties order.
+
+The `order` config enforces a consistent order of content in your declaration blocks:
+
+1. SASS variables,
```

Comment:
Actually it's "Sass" (and "SCSS")

https://sass-lang.com/

Reply:
Ok, I have changed this also in our Textlint config package, so only the `Sass` term is allowed.